src/config.rs

```diff
@@ -116,22 +119,57 @@ impl Default for SortValue {
     }
 }
 impl Config {
+    /// Set timezone on Config struct only
+    pub fn with_timezone(self: &Config, timezone: &str) -> Config {
+        Config {
+            timezone: Some(timezone.to_string()),
+            ..self.clone()
+        }
+    }
+
+    pub async fn projects(self: &Config) -> Result<Vec<Project>, Error> {
```

Comment:
- [ ] Comment that it pulls in new projects if there are only legacy projects and leave link to ticket to remove.